service/geopmdpy/access.py

```diff
@@ -392,10 +422,19 @@ def main():
                                  help='Do error checking on all user input, but do not modify configuration files')
     parser_group_nF.add_argument('-F', '--force', dest='force', action='store_true', default=False,
                                  help='Write access list without validating GEOPM Service support for names')
+    parser.add_argument('-x', '--direct', action='store_true', help='Write directly to files, do not use DBus')
+
     args = parser.parse_args()
+    if args.direct and not (args.group or args.default):
+        # This option is intended for early admin access list management before
+        # the GEOPM service is running. The option does not have a meaningful
+        # use case when run for the root user.
+        parser.error('Must specify either --group or --default with --direct.')
+
     try:
-        acc = Access(SystemMessageBus().get_proxy('io.github.geopm',
-                                                  '/io/github/geopm'))
+        geopm_proxy = (None if args.direct
```

Comment:
As an alternative to `None`, we could add a `DirectAccessProxy` that implements the proxy interface but makes direct calls. Doing so would mean we can avoid changes to the `TestAccess.py` file

Reply:
That would also remove all of the other `if none` checks throughout this file

Reply:
that seems like a cleaner approach.  I think we can wait until after monday for this change.  Thanks!